<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailFIlmWidget</class>
 <widget class="QDialog" name="DetailFIlmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>1452</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>234324</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #23282d;</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>49</y>
     <width>1151</width>
     <height>1391</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::WheelFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::VLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1134</width>
      <height>1391</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>891</width>
       <height>651</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	background-color:#161a1f;
	border-radius: 10px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="poster_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>240</width>
        <height>360</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/SAFA/Downloads/pobeg.jpeg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="title_label">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 22pt &quot;Segoe UI&quot;;
background-color:#161a1f;
color:white;</string>
      </property>
      <property name="text">
       <string>Бойцовский клуб 18+</string>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::RichText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="duration_label">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>84</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color:#161a1f;
font: 700 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Время</string>
      </property>
     </widget>
     <widget class="QLabel" name="year_label">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>145</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color:#3c434b;
padding:5px 20px 5px 6px;
border-radius:5px;
font: 700 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>2018</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="genre_label">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>202</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color:#ff5858;
padding:5px 20px 5px 0px;
border-radius:5px;
font: 700 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Жанр</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>12</number>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="director_label">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>267</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
text-align:center;
background-color:#3c434b;
padding:5px 20px 5px 6px;
border-radius:5px;
font: 700 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Режиссёр</string>
      </property>
     </widget>
     <widget class="QLabel" name="description_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>446</y>
        <width>821</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#161a1f;
font: 11pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Сотрудник страховой компании страдает хронической бессонницей и отчаянно пытается вырваться из мучительно скучной жизни. Однажды в очередной командировке он встречает некоего Тайлера Дёрдена – харизматического торговца мылом с извращенной философией. Тайлер уверен, что самосовершенствование – удел слабых, а единственное, ради чего стоит жить, – саморазрушение. Проходит немного времени, и вот уже новые друзья лупят друг друга почем зря на стоянке перед баром, и очищающий мордобой доставляет им высшее блаженство. Приобщая других мужчин к простым радостям физической жестокости, они основывают тайный Бойцовский клуб, который начинает пользоваться невероятной популярностью.</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="title_label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 20pt &quot;Segoe UI&quot;;
background-color:#161a1f;
color:white;</string>
      </property>
      <property name="text">
       <string>Описание</string>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::RichText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="rating_label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>91</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray;
background-color:#161a1f;
font: 700 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Время</string>
      </property>
     </widget>
     <widget class="QLabel" name="year_label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>151</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray;
background-color:#161a1f;
font: 700 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Год</string>
      </property>
     </widget>
     <widget class="QLabel" name="genre_label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>206</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray;
background-color:#161a1f;
font: 700 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Жанр</string>
      </property>
     </widget>
     <widget class="QLabel" name="director_label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>268</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray;
background-color:#161a1f;
font: 700 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Режиссёр</string>
      </property>
     </widget>
     <widget class="QLabel" name="duration_label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>334</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray;
background-color:#161a1f;
font: 700 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Рейтинг</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>532</x>
        <y>334</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
text-align:center;
background-color:#3c434b;
padding:5px 20px 5px 0px;
border-radius:5px;
font: 700 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>2</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
padding:0;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/SAFA/Downloads/d68d368b1fdb1aa9448170f4b8d35a3c.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="rating_label">
       <property name="geometry">
        <rect>
         <x>43</x>
         <y>1</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;
text-align:center;
background-color:#3c434b;
border-radius:5px;
font: 700 15pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Рейтинг</string>
       </property>
      </widget>
     </widget>
     <zorder>title_label_3</zorder>
     <zorder>poster_label</zorder>
     <zorder>title_label</zorder>
     <zorder>duration_label</zorder>
     <zorder>year_label</zorder>
     <zorder>genre_label</zorder>
     <zorder>director_label</zorder>
     <zorder>rating_label_3</zorder>
     <zorder>year_label_3</zorder>
     <zorder>genre_label_3</zorder>
     <zorder>director_label_3</zorder>
     <zorder>duration_label_3</zorder>
     <zorder>description_label</zorder>
     <zorder>groupBox_4</zorder>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>710</y>
       <width>867</width>
       <height>551</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#161a1f;
border-radius:10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton_backward">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>485</y>
        <width>71</width>
        <height>57</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#161a1f;
border-radius:0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_play_pause">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>485</y>
        <width>71</width>
        <height>57</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#161a1f;
border-radius:0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_forward">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>485</y>
        <width>71</width>
        <height>57</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#161a1f;
border-radius:0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Volume">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>485</y>
        <width>61</width>
        <height>57</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#161a1f;
border-radius:0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_Volume">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>501</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>0</y>
        <width>868</width>
        <height>471</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_Video">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>0</y>
         <width>863</width>
         <height>441</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox{
	border-radius:15px;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_current_time">
       <property name="geometry">
        <rect>
         <x>739</x>
         <y>445</y>
         <width>55</width>
         <height>20</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTipDuration">
        <number>1</number>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Segoe UI&quot;;
font-weight:700;
background-color:#161a1f;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string>0:00:00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_end_time">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>445</y>
         <width>55</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Segoe UI&quot;;
font-weight:700;
background-color:#161a1f;</string>
       </property>
       <property name="text">
        <string>0:00:00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>796</x>
         <y>447</y>
         <width>9</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Segoe UI&quot;;
font-weight:700;
background-color:#161a1f;</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_Duration">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>446</y>
         <width>711</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#161a1f;
border-radius:0px;
cursor: pointer;

</string>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <zorder>groupBox_Video</zorder>
      <zorder>label_end_time</zorder>
      <zorder>label_2</zorder>
      <zorder>horizontalSlider_Duration</zorder>
      <zorder>label_current_time</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>6</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                               border-radius:15px;
                               background-color:#161a1f;}
QPushButton:hover{
                               background-color:#4b4b4b;
                               border:1px solid #303030;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>back_arrow.png</normaloff>back_arrow.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
